studio/schemas/objects/sections/imagesplit.ts

```diff
@@ -0,0 +1,38 @@
+import { defineField } from "sanity";
+
+import { imageExtended } from "studio/schemas/fields/media";
+import { richText, title } from "studio/schemas/fields/text";
+import { link } from "studio/schemas/objects/link";
+
+export const imageID = "imageSplitSection";
+
+export const imageSplitSection = defineField({
+  name: imageID,
+  title: "Image Split",
+  type: "object",
+  fields: [
+    title,
+    richText,
```

Comment:
This might be a problem for another PR, but does richText support different languages? As far as I can see it does not, and we need the content available in norwegian/swedish and english.

And the title should also be internationalized

Reply:
We can just make it non-rich-text

Reply:
Yeah i think that would be best! It is in line with what we have done on the customer case page, which was to use plain internationalized text to force the correct styling and to prevent content creators from adding their own headings in the text.